### PR TITLE
tagutil: init at 3.1

### PR DESCRIPTION
**Pull request by dan4ik605743**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by dan4ik605743 — COMMENTED** [review]
image
Package checked, everything works.

File: pkgs/applications/audio/tagutil/default.nix [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    cmake
+    pkg-config
+  ];
+
+  buildInputs = [
+    libvorbis
+    libyaml
+    jansson
+    taglib
+    zlib
```
Comment (fabaff) [inline pkgs/applications/audio/tagutil/default.nix:32]: Would adding [`flac`](https://search.nixos.org/packages?channel=unstable&show=flac&from=0&size=50&sort=relevance&query=flac) enable support for this format?

File: pkgs/applications/audio/tagutil/default.nix [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  ];
+
+  meta = with lib; {
+    description = "Scriptable music files tags tool and editor.";
```
Comment (musfay) [inline pkgs/applications/audio/tagutil/default.nix:36]:
```suggestion
    description = "Scriptable music files tags tool and editor";
```

File: pkgs/applications/audio/tagutil/default.nix [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  src = fetchFromGitHub {
+    owner = "kaworu";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-oY1aGl5CKVtpOfh8Wskio/huWYMiPuxWPqxlooTutcw=";
+  };
+
+  cmakeFlags = [
+    "../src"
+  ];
```
Comment (musfay) [inline pkgs/applications/audio/tagutil/default.nix:20]:
```suggestion
  sourceRoot = "source/src";
```

**Review by musfay — CHANGES_REQUESTED** [review]

File: pkgs/applications/audio/tagutil/default.nix (hunk 1 above)
Comment (dan4ik605743) [reply pkgs/applications/audio/tagutil/default.nix:32]: There is a problem with the code, in the future it is possible to solve.

File: pkgs/applications/audio/tagutil/default.nix (hunk 2 above)
[the remark this answers is not in the document]
Comment (dan4ik605743) [reply pkgs/applications/audio/tagutil/default.nix:36]: ok

File: pkgs/applications/audio/tagutil/default.nix (hunk 3 above)
Comment (dan4ik605743) [reply pkgs/applications/audio/tagutil/default.nix:20]: ok

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 134117` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>tagutil</li>
  </ul>
</details>

**Review by fabaff — APPROVED** [review]
- [x] package path fits guidelines
- [x] package name fits guidelines
- [x] package version fits guidelines
- [x] `meta.description` is set and fits guidelines
- [x] `meta.license` fits upstream license
- [x] `meta.maintainers` is set
- [x] package builds (see https://github.com/NixOS/nixpkgs/pull/134117#issuecomment-901489791)
- [ ] all depending packages build
- [x] tested on x86_64

```bash
...ew/pr-134117]$ ./results/tagutil/bin/tagutil -h
tagutil v3.1

usage: tagutil [OPTION]... [ACTION:ARG]... [FILE]...
Modify or display music file's tag.
[...]
```

Looks good to me.

**Pull request merged**